cluster-operator/src/main/java/io/strimzi/operator/cluster/model/ZookeeperConfiguration.java

```diff
@@ -7,34 +7,25 @@
 
 import io.strimzi.api.kafka.model.ZookeeperClusterSpec;
 
-import java.util.ArrayList;
-import java.util.Arrays;
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 
-import static java.util.Collections.emptyList;
+import static java.util.Arrays.asList;
 
 /**
  * Class for handling Zookeeper configuration passed by the user
  */
 public class ZookeeperConfiguration extends AbstractConfiguration {
 
     private static final List<String> FORBIDDEN_OPTIONS;
+    private static final List<String> EXCEPTIONS;
     protected static final Map<String, String> DEFAULTS;
 
     static {
-        FORBIDDEN_OPTIONS = new ArrayList<>();
-        FORBIDDEN_OPTIONS.addAll(Arrays.asList(ZookeeperClusterSpec.FORBIDDEN_PREFIXES.split(" *, *")));
-        // This option is handled in the Zookeeper container startup script
-        FORBIDDEN_OPTIONS.add("snapshot.trust.empty");
-        // This option would prevent scaling beyond 1 node for clusters started with a single node
-        FORBIDDEN_OPTIONS.add("standaloneEnabled");
-        // Reconfiguration needs to be enabled to allow scaling of the cluster
-        FORBIDDEN_OPTIONS.add("reconfigEnabled");
-        // The Cluster Operator requires access to multiple 4LW and access to the nodes is secured by the TLS-Sidecars so we set all allowed
-        FORBIDDEN_OPTIONS.add("4lw.commands.whitelist");
+        FORBIDDEN_OPTIONS = asList(ZookeeperClusterSpec.FORBIDDEN_PREFIXES.split(", "));
+        EXCEPTIONS = asList(ZookeeperClusterSpec.FORBIDDEN_PREFIX_EXCEPTIONS.split(", "));
```

Comment:
`FORBIDDEN_OPTIONS_EXCEPTIONS`  might be a slightly better name just so that this isn't confused with `EXCEPTIONS` as in the throwable (highly unlikely I admit, but still probably best to stay consistent with where  we have used:
`FORBIDDEN_PREFIXES`
`FORBIDDEN_PREFIXES_EXCEPTIONS`

Reply:
I think this is fair point, but it is named `EXCEPTIONS` everywhere else - in all the other *Configuration classes in the`model` package. So I probably prefer to keep it as it is and in sync with the other classes. I would instead open a sepaarte issue for it if it is fine with you.

Reply:
I actually decided to go ahead and opened #2927 - I think it makes nice and easy issue for someone to get started with it. I hope you don't mind.

Reply:
Makes sense to me, thanks for raising an issue for it - leaving as is absolutely fine in my eyes :)